src/Bicep.Cli/Services/CompilationService.cs

```diff
@@ -107,8 +110,12 @@ public async Task<DecompileResult> DecompileAsync(string inputPath, string outpu
             inputPath = PathHelper.ResolvePath(inputPath);
             Uri inputUri = PathHelper.FilePathToFileUrl(inputPath);
             Uri outputUri = PathHelper.FilePathToFileUrl(outputPath);
+            if (!fileResolver.TryRead(inputUri).IsSuccess(out var jsonContents, out _))
```

Comment:
nit: Can omit `out _`
```suggestion
            if (!fileResolver.TryRead(inputUri).IsSuccess(out var jsonContents))
```